Sources/SwiftGRPC/Core/Channel.swift

```diff
@@ -124,41 +124,43 @@ private extension Channel {
     }
 
     deinit {
-      completionQueue.shutdown()
+      shutdown()
     }
 
     private func run() {
       let spinloopThreadQueue = DispatchQueue(label: "SwiftGRPC.ConnectivityObserver.run.spinloopThread")
 
       spinloopThreadQueue.async { [weak self] in
```

Comment:
No need to use `[weak self]` here, as mentioned before. Shutdowns are initiated by `Channel`.